### PR TITLE
🧹 simplify docs

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: apps/cnquery/cmd/run.go [hunk 1]
```diff
@@ -23,15 +23,15 @@ func init() {
 
 var execCmd = builder.NewProviderCommand(builder.CommandOpts{
 	Use:   "run",
-	Short: "Run a Mondoo Query Language (MQL) query",
-	Long:  `Run a Mondoo Query Language (MQL) query on the CLI and displays its results.`,
+	Short: "Run a MQL query",
```
Comment (misterpantz) [inline apps/cnquery/cmd/run.go]: please change this line and the next line to read "an MQL"

**Review by misterpantz — CHANGES_REQUESTED** [review]
Just a couple of changes, please

File: apps/cnquery/cmd/run.go (hunk 1 above)
Comment (chris-rock) [reply apps/cnquery/cmd/run.go]: Thank you, fixed

**Review by imilchev — APPROVED** [review]

File: resources/packs/azure/azure_rm_web.go [hunk 2]
```diff
@@ -122,7 +122,7 @@ func (a *mqlAzurermWeb) GetAvailableRuntimes() ([]interface{}, error) {
 
 	res := []interface{}{}
 	windows := web.Enum15Windows
-	// NOTE: we do not return a MQL resource since stacks do not have their own proper id in azure
+	// NOTE: we do not return MQL resource since stacks do not have their own proper id in azure
```
Comment (misterpantz) [inline resources/packs/azure/azure_rm_web.go:125]: an  MQL resource

Comment (misterpantz) [inline resources/packs/azure/azure_rm_web.go:125]: return MQL resource --> return an MQL resource

**Pull request merged**